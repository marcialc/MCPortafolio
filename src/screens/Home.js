import React, { Component } from "react";
import Data from "../Data";

export default class Home extends Component {
  render() {
    return (
      <React.Fragment>
        <header id="home">
          <nav id="nav-wrap">           
            <ul id="nav" className="nav">
              <li className="current">
                <a className="smoothscroll" href="#home">
                  Home
                </a>
              </li>
              <li>
                <a className="smoothscroll" href="#about">
                  About
                </a>
              </li>
              <li>
                <a className="smoothscroll" href="#experience">
                  Experience
                </a>
              </li>
            </ul>
          </nav>
          <div className="row banner">
            <div className="banner-text">
              <h1 className="responsive-headline">Marcial Cabrera</h1>
              <h3>Software Engineer / Front-end Developer / Mobile Developer</h3>
              <ul className="social">
                {Data.socialLinks &&
                  Data.socialLinks.map(item => {
                    return (
                      <li key={item.name}>
                        <a href={item.url}>
                          <i className={item.className} />
                        </a>
                      </li>
                    );
                  })}
              </ul>
            </div>
          </div>
        </header>
      </React.Fragment>
    );
  }
}
