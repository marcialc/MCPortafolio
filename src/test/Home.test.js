import React from 'react';
import Home from '../screens/Home';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

configure({adapter: new Adapter()});

describe('React component test with Enzyme', () => {
  it('renders home crashing', () => {
     shallow(<Home.name/>);
   });
});


test('name', () => {
    const name = Home.h1;
    console.log(name)
    expect(name).not.toBeNull()
})