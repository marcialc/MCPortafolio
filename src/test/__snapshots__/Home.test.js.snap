// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter snapshot renders 1`] = `
<nav
  id="nav-wrap"
/>
`;

exports[`data 1`] = `
Object {
  "default": [Function],
}
`;

exports[`wrapper to nav bar snapshot renders 1`] = `
<nav
  id="nav-wrap"
/>
`;
